docs/release-notes/2.32.0.md

```diff
@@ -0,0 +1,166 @@
+- [PMM-10502](https://jira.percona.com/browse/PMM-10502), [PMM-10831](https://jira.percona.com/browse/PMM-10831): [Backup Management]: Improved User Experience and labels on all four tabs of the Backup section: 
+  - added icons to better illustrate failed and sucessful backups.
+  - added **Service Name** column to show the service from where the backup was retrieved.
+  - renamed **DB Vendor** tab to **DB Technology**.
+  - improved **Actions** menu with clearer labelled options.
+
+- [PMM-10742](https://jira.percona.com/browse/PMM-10742): [Backup Management]: Extended PMM compatibility with Percona Backup for MongoDB (PMB) to support the latest version, v. 2.0 PBM.
+- [PMM-10829](https://jira.percona.com/browse/PMM-10829): [Backup Management]: The backups table under Backup > All Backups now includes a new Location column which displays the service where the backup came from (S3 or Local Storage).
+
+ 
+- [PMM-8792](https://jira.percona.com/browse/PMM-8792), [PMM-9055](https://jira.percona.com/browse/PMM-9055), [PMM-9056](https://jira.percona.com/browse/PMM-9056): PMM 2.32.0 introduces the parameter `--max-query-length` in the `pmm-admin` command for MYSQL, PostgreSQL, and MongoDB databases. Now, queries longer than 2048 characters are not truncated (if parameter is set to `-1`, which means no limit or value greater than 2048 chars), and the **Explain** and **Examples** tabs on the QAN dashboard do not throw an error, see [PMM-1899](https://jira.percona.com/browse/PMM-1899). These tabs are important as they provide contextual information pertaining to the query.
+
+- [PMM-9545](https://jira.percona.com/browse/PMM-9545): To monitor Kubernetes clusters, PMM now collects Kubernetes metrics from kube-state-metrics. For more information on kube-state-metrics, see  [documentation](https://github.com/kubernetes/kube-state-metrics).
+    
+    !!! caution alert alert-warning "Disclaimer"
+        This feature is still [technical preview](../details/glossary.md#technical-preview) and is subject to change. We recommend that early adopters use this feature for testing purposes only.
+
+- [PMM-10579](https://jira.percona.com/browse/PMM-10579): Only monitored services are displayed on the main menu for a clean and simple User Interface.
+- [PMM-10512](https://jira.percona.com/browse/PMM-10512): [Alerting]:  When creating new alert rules. you can now use two new labels (**application_name** and **usename**) for **pg_stat_activity_** metrics.
+
+
+## Improvements
+
+  - [PMM-10156](https://jira.percona.com/browse/PMM-10156): Starting with PMM 2.32.0, we have an enhanced PMM dashboard that gives a bird-view of the services, infrastructure, and critical issues (if any). Thus, you can effortlessly diagnose the problem from the home dashboard with just one click.
+
+  
+- [PMM-9946](https://jira.percona.com/browse/PMM-9946): To enhance user experience, we have simplified the deployment of PMM inside the K8s cluster using Manifest or Helm.
```

Comment:
To enhance user experience, once PMM is deployed to a Kubernetes cluster and a user wants to use DBaaS, PMM automatically registers that Kubernetes cluster to DBaaS.